apps/client/components/Post/detail/Likes/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { css, useTheme } from "@emotion/react";
+import { Flex, padding, Stack } from "@toss/emotion-utils";
+import Image from "next/image";
+import { PercentBar } from "~/components/Common";
+import { Text } from "~/components/Common/Typo";
+import { useFetchLikePercent } from "~/queries/restaurant";
+import { countryToSvg } from "~/utils/country";
+
+interface Props {
+  id: string;
+}
+const Likes = ({ id }: Props) => {
+  const { colors } = useTheme();
+
+  const { data } = useFetchLikePercent(+id);
+  return (
```

Comment:
```suggestion
  const { colors } = useTheme();
  const { data } = useFetchLikePercent(+id);
  
  return (
```

기존 코드 스타일로 변경해주시면 감사합니다!